example_config.json

```diff
@@ -34,9 +35,6 @@
 				"specjbb.comm.connect.timeouts.read": 650000,
 				"specjbb.comm.connect.timeouts.write": 650000,
 				"specjbb.comm.connect.worker.pool.max": 320,
-				"specjbb.customerDriver.threads": 64,
-				"specjbb.customerDriver.threads.saturate": 144,
-				"specjbb.customerDriver.threads.probe": 96,
```

Comment:
We should make a PR to update these props, they aren't ever used in the v1 for SPECjbb2015. Remind me to make an issue about that later.

Reply:
Yeah, it causes problems